database/seeds/October16NewCategoryTableSeeder.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+use Carbon\Carbon;
```

Comment:
Seems good. Thanks for PR @sahilsharma011 

Going forward, please create a new issue for such things or you could have reopen previous issue. **My bad** that I didn't tested it till now and as result, #137 is still not live. Good you tested and find the issue, thanks a lot.

Seems its high time to implement PHP unit & CI server.

OOPs, you also send changes from your master branch. I'll merge this time but in future, I recommend not to do it as if PR is not merged, you will find it difficult to get changes in master branch and merge it to other issue branches. Always have `master -> development -> issue-xyz` structure and send PR from `issue-xyz` to `development` branch.

No action needed this time. I'll merge is for now so that you can continue with your master branch. Thanks again for finding a bug and fixing it.

@aabi01 you forgot this in your PR. Didn't you tested it before creating PR? Please always test your code before sending PR. Lesson to me: always test before merging.

For not, no actions for anyone, I'll merge it and test it once again before going live.